### ISSUE TITLE
Move this website to Netlify

### ISSUE THREAD

**Issue by bassam** [crossplane/crossplane.github.io#98]
We should host this website on Netlify like we do with other sites.

**Comment by sftim** [comment]
Want help migrating? I might be able to offer advice.

**Comment by bassam** [comment]
we should also consider using netlify to build different branches of the site, and simplify our documentation versioning story. Specifically, each branch including master would have *one* version of the documentation, and then we use netlify to build the multiple sites (one for each supported branch). We suspect this will improve the SEO and indexing story as well.

**Comment by bassam** [comment]
@sftim we'd love help if you're up for it.

**Comment by jbw976** [comment]
We've made a lot of progress on this recently and there are a few moving parts, so I wanted to write-up the current plan for more visibility.  @plumbis and @jbw976 will keep driving this with support from CNCF ServiceDesk and LF IT.

### TL;DR:

1. Move DNS hosting from LF IT to Netlify so we have self-service DNS management
2. Move crossplane.io website content to Vercel
3. Move crossplane docs content to Netlify

### Detailed rollout plan:

* We're taking the step now/asap to move DNS ownership/management from LF IT to Netlify.  The CNCF has given us access to Netlify, so when the ownership move is complete we will have self-service access to updating DNS entries as we need.
* After this one (big) change is done, we will verify everything is still working and make any fixes to DNS if needed via the self-service.  There will be **no** _content hosting_ updates yet at this point.
* If everything is OK with the new DNS and current content hosting, then we'll start **content hosting** transfers.
* Main site content of crossplane.io and www.crossplane.io will be moved over to Vercel where the Unomena folks have their site refresh ready.
  * The source code for this will be in https://github.com/crossplane/website.
  * this site will have a redirect to point `www.crossplane.io/docs/**`--> `docs.crossplane.io/**`, which will point to our existing docs in Github pages
* @plumbis will keep updating the docs content hosted in Netlify (crossplane.netlify.app) and get it polished and finalized
* We'll update DNS for docs.crossplane.io to @plumbis new docs in Netlify

**Comment by plumbis** [comment]
The Netlify builds before deploy point to the [theme-update](https://github.com/crossplane/crossplane.github.io/tree/theme-update) branch of this repo. When we "switch over" Netlify will point to the `master` branch of this repo. 

Updating the [name of the repo](https://github.com/crossplane/crossplane.github.io/issues/152) will be a separate task to deal with in the future.

**Comment by plumbis** [comment]
docs.crossplane.io is now fully on netlify. closing as completed.
